src/assets/cru-scss/_subnav.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
```

Comment:
You using math here?

Reply:
Hmm I must've had a reason for adding it, but I thought I only added it to files that I was removing the `/` from.  Like you said, it was a lot of CSS to go through!  I'll remove it.